### PR TITLE
Update Ignition to Gz

### PR DESCRIPTION
**Pull request by MarqRazz**
This PR updates all instances of Ignition to Gz to support newer versions of Gazebo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Format Status](https://github.com/MarqRazz/realsense2_gz_description/actions/workflows/format.yaml/badge.svg)](https://github.com/MarqRazz/realsense2_gz_description/actions/workflows/format.yaml)
 
-Description: This ROS 2 package is designed to be used in unison with [realsense2_description](https://github.com/IntelRealSense/realsense-ros/tree/ros2-master/realsense2_description) and allows for easy definition of Realsense cameras that can be simulated in Gazebo Fortress and newer. It may support other versions of Ignition Gazebo but this has not been tested.
+Description: This ROS 2 package is designed to be used in unison with [realsense2_description](https://github.com/IntelRealSense/realsense-ros/tree/ros2-master/realsense2_description) and allows for easy definition of Realsense cameras that can be simulated in Gazebo Fortress and newer. It may support other versions of Gazebo but this has not been tested.
```
Comment (forrest-rm) [inline README.md:5]: Is `Gazebo Fortress` still the correct version that is supported?

**Review by forrest-rm — APPROVED** [review]
Given that this is open source, do we want to create a `humble` branch from `main` before this change?

File: README.md (hunk 1 above)
Comment (MarqRazz) [reply README.md:5]: Yes Fortress is used for Humble and Iron.

**Comment by MarqRazz** [comment]
Tested and confirmed that these changes are supported in Humble and Iron :confetti_ball:

**Review by bmagyar — APPROVED** [review]
Thank you!

**Pull request merged**